turbo.json

```diff
@@ -17,12 +19,20 @@
     },
     "docs#build": {
       "dependsOn": ["^build"],
-      "env": ["NEXT_PUBLIC_*", "NODE_ENV", "OPENAI_API_KEY", "SKIP_BUILD_STATIC_GENERATION"],
+      "env": [
+        "ANALYZE",
+        "NEXT_PUBLIC_*",
+        "NODE_ENV",
+        "OPENAI_API_KEY",
+        "SKIP_BUILD_STATIC_GENERATION",
+        "SUPABASE_SERVICE_ROLE_KEY"
```

Comment:
this one was removed from docs cause we don't need it

Reply:
I added it cause of the scripts, but you're right, those scripts are not related to the `build` command. Good catch!